File: py/problem_2.py
# Problem 2
def sumOfFibonacciEven(n):
	x,y = 1,2
	sum = 0
	while x < n:
		x,y = y,x+y
		if x % 2 == 0:
			sum += x

	return sum

if __name__ == "__main__":
	sum1 = sumOfFibonacciEven(4e6)
	print("The sum of the Fibonacci numbers not exceeding 4 Million {}".format(sum1))

File: js/problem_2.js
function sumOfFibonacciEven (n) {
    let x = 1;
    let y = 2;
    let sum1 = 0;

    while (x < n) {
        x = y;
        y = x + y;
        if (x % 2 === 0) {
            sum1 += x;
        }
    }
    return {sum1};
}

console.log(sumOfFibonacciEven(4e6));